src/components/ArrowLink/ArrowLink.test.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { render, cleanup } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+
+import ArrowLink, { TEST_ID } from './ArrowLink';
+
+const renderComponent = () => {
+	return render(<ArrowLink>Arrow Link</ArrowLink>);
+};
+
+afterEach(() => {
+	cleanup();
+	jest.clearAllMocks();
+});
+
+describe('ArrowLink componnent', () => {
```

Comment:
Tiny nitpick: there's a small spelling error here - component should only have one n (component vs componnent)